README.md

```diff
@@ -13,29 +13,48 @@ The toolchain includes the following:
 - `dojo-bind`: Generate bindings for various languages / frameworks (typescript, phaser / rust, bevy).
 
 ## Development
-### Prerequisites
-- Install [Rust](https://www.rust-lang.org/tools/install)
-- Setup Rust:
+
+### Setup Submodules
+
 ```
-rustup override set stable && rustup update && cargo test
+git submodule update --init --recursive
 ```
 
-### Using the language server
+## Development container
+
+It is recommended to use the dev container when building on DoJo as it contains everything needed to begin developing.
```

Comment:
Can you add a link to [here](https://github.com/microsoft/vscode-dev-containers/blob/main/containers/rust/README.md) or better yet, provide an example how to use the `bullseye` variant for Apple Silicon HW? I'm not that familiar with dev containers, so this would help 🙏

Reply:
yes, we can add it into a .env var pre-build. Ill do this